e2e/cypress/integration/bot_accounts/crud_spec.js

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @bot_accounts
+
+import {getRandomId} from '../../utils';
+
+describe('Bot accounts - CRUD Testing', () => {
+    let newTeam;
+    let botName;
+    let botDescription;
+    beforeEach(() => {
+        botName = 'bot-' + Date.now();
+        botDescription = 'test-bot-' + Date.now();
+
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+                DisableBotsWhenOwnerIsDeactivated: true,
+            },
+            PluginSettings: {
+                Enable: true,
+                RequirePluginSignature: false,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Create a test bot
+        cy.apiCreateBot(botName, 'Test Bot', botDescription);
+
+        // # Create and visit new channel
+        cy.apiInitSetup().then(({team}) => {
+            newTeam = team;
+        });
+        cy.apiAdminLogin();
+    });
+
+    it('MM-T1841 Long description text', () => {
+        // # Visit the integrations
+        cy.visit(`/${newTeam.name}/integrations/bots`);
+
+        // * Check that the previously created bot is listed
+        cy.findByText(`Test Bot (@${botName})`).then((el) => {
+            // # Make sure it's on the screen
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
```

Comment:
Is there no better way to find the element? Using parent can lead to brittle tests.

Reply:
not that I am aware of :(